ledfx/virtuals.py

```diff
@@ -357,6 +389,29 @@ def set_effect(self, effect):
                 _LOGGER.warning(error)
                 raise ValueError(error)
 
+            if fallback:
+                _LOGGER.warning("Fallback requested")
+                if self._active_effect is not None:
+                    if self.last_active_effect_type is None:
+                        self.last_active_effect_type = self._active_effect.type
+                        _LOGGER.warning(
+                            f"Setting fallback to {self.last_active_effect_type}"
+                        )
+                    else:
+                        # TODO: this will be a problem, how do you clear a fallback that has not triggered, needs some thought
+                        _LOGGER.warning(
```

Comment:
I can get rid of this comment. It can be cleared with setting any effect with fallback false or just not specified. Which is the behavior we want.